seabird_chat_ingest.proto

```diff
@@ -114,6 +120,11 @@ message ChangeChannelEvent {
 //
 // HelloChatEvent is the only message type that is required for a chat backend
 // to work.
+//
+// Core may be configured with both time and load-based request failure
```

Comment:
`...both time- and...`

Reply:
I've never seen this syntax used before... maybe `both time-based and load-based` would make sense?